server/src/main/java/org/jetbrains/bsp/bazel/server/sync/languages/EmptyLanguagePlugin.java

```diff
@@ -12,4 +14,10 @@ public Option<LanguageData> resolveModule(BspTargetInfo.TargetInfo targetInfo) {
 
   @Override
   protected void applyModuleData(LanguageData moduleData, BuildTarget buildTarget) {}
+
+  @Override
+  public Option<Path> calculateSourceRoot(Path source) {
```

Comment:
probably u can just return Option.none(), since it's only an empty language plugin

Reply:
Changed to Option.none()